sdk/examples/wallet/spammer.rs

```diff
@@ -39,23 +44,36 @@ async fn main() -> Result<()> {
     // Restore wallet from a mnemonic phrase.
     let client_options = ClientOptions::new().with_node(&std::env::var("NODE_URL").unwrap())?;
     let secret_manager = MnemonicSecretManager::try_from_mnemonic(std::env::var("MNEMONIC").unwrap())?;
+
+    // TODO: in this case we can just let the builder generate the wallet address ... so remove?
```

Comment:
Would keep it, not sure if we want to keep the address generation in the builder like it is